.gcb/integration.yaml

```diff
@@ -40,7 +40,7 @@ steps:
       #!/usr/bin/env bash
       set -e
       cargo build
-      cargo test --package integration-tests --features run-integration-tests
+      cargo test --package integration-tests --features run-integration-tests --features log-integration-tests pubsub
```

Comment:
pubsub only? :monocle_face:

Reply:
This was for testing :) Already backed out